website/content/tutorial/productivity/notion.mdx

```diff
@@ -4,6 +4,19 @@ title: Agentica > Tutorial > Notion Agents
 
 ## Introduction
 
+<br />
+<iframe
+  src="https://www.youtube.com/shorts/hzkNNt7bvUQ"
```

Comment:
```suggestion
  src="https://www.youtube.com/embed/hzkNNt7bvUQ"
```

please use embed link
short link cause cors error

Reply:
nononoo already modified it

thank you for your attention

Reply:
Yes, I checked